tests/test_examples.py

```diff
@@ -86,6 +86,9 @@ def file_cmp(file1, file2, filter_string=None):
                 if first:
                     left  = re.sub('[0-9]{4}\\\\-[0-9]{2}\\\\-[0-9]{2}', '!!DATE!!', left)
                     right = re.sub('[0-9]{4}\\\\-[0-9]{2}\\\\-[0-9]{2}', '!!DATE!!', right)
+                    # issue #50, setuptools v60+
+                    right = re.sub(".dev0", "-dev", right)
```

Comment:
It should be:
```diff
-                    right = re.sub(".dev0", "-dev", right)
+                    right = re.sub("-dev", ".dev0", right)
```

```console
python3 -c 'from packaging.version import Version; print(Version("0.1.0-dev"))'
0.1.0.dev0
```
```

Reply:
Also consider that dot has a special meaning in regular expressions, so if you indeed intend to replace ".dev0" with "-dev", consider *not* using regular expressions, but basic string replace method.